### PR TITLE
[REF] 8.0 document page environment manual

### PR DESCRIPTION
**Pull request by llacroix**
Marked as installable

Depends on:
- [x] #23

### CODE REVIEW

File: document_page_environment_manual/__openerp__.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "depends": ['mgmtsystem_manuals'],
     "data": ['document_page_environment_manual.xml'],
     "demo": [],
-    'installable': False,
+    'installable': True,
     "certificate": ''
 }
 # vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
```
Comment (bwrsandman) [inline document_page_environment_manual/__openerp__.py]: Please remove vim lines

**Comment by bwrsandman** [comment]
Please rename `document_page_environment_manual.xml` to `document_page_data.xml`

**Comment by bwrsandman** [comment]
Depends on #23

**Comment by max3903** [comment]
IOError: File not found: document_page_environment_manual/document_page_environment_manual.xml

**Comment by max3903** [comment]
Tested on runbot.

:+1:

**Comment by dreispt** [comment]
:+1:

**Pull request merged**